### ISSUE TITLE
[Keyboard Navigation – Make Code - Control Pane] Information is not accessible which is under button controls.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3671]
User Experience:
If controls are not accessible while navigating through keyboard, it becomes an issue for keyboard dependent users to identify and access the control and hence miss the important information related to the controls and functionality.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Basic, Input, Music, Led, more, Radio" controls and select it.
6. Navigate through all the controls appearing after selection of element.
7. verify the issue

Actual Result:
Information is not accessible which is under button controls.

Expected Result:
All the content/Information should be accessible using keyboard as well as mouse which is available under screen.

Note:
- This issue also exists with Firefox and Chrome Browser.
- This issue exists throughout the application
- Same issue exists under "Edit Function" dialog

WCAG Reference:
[WCAG 2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.1.1, #Win10-Edge(Chromium)

[Uploading WCAG2.1.1_Not accessible.zip…]()

**Comment by Chandangithb** [comment]
As discussed with internal SME severity of the bug is correct. #A11ySev_Reviewed

**Comment by testingmakecode** [comment]
As checked on [Microsoft Make Code Arcade] (https://arcade.makecode.com/) the issue is still repro.

**Comment by testingmakecode** [comment]
As checked on [Microsoft Make Code Arcade] (https://arcade.makecode.com/) the issue is still repro.

**Comment by srietkerk** [comment]
We have an exception for the accessibility of blocks.
